androidshared/src/main/java/org/odk/collect/androidshared/ui/ToastUtils.kt

```diff
@@ -17,6 +17,10 @@ import org.odk.collect.strings.localization.getLocalizedString
  */
 object ToastUtils {
 
+    @JvmStatic
+    var recordToasts = false
```

Comment:
I looked at pulling out an interface that we can inject a recording fake of instead of using static state like this, but it felt complicated by the middle toast functionality. I think we can revise this when we look at reworking constraints for Android 11.